client/src/components/Login/index.js

```diff
@@ -37,49 +37,54 @@ class Login extends Component {
     });
   };
 
-  render() {
-    return (
-      <React.Fragment>
-        <Title />
-        <Header title="Login to save your data!" />
-        <StyledForm>
-          <StyledLabel> Username* </StyledLabel>
-          <StyledInput
-            type="text"
-            name="username"
-            placeholder="username"
-            value={this.state.username}
-            onChange={e => this.setState({
-              username: e.target.value,
-            })
+goSignUp = () => {
+  const { history } = this.props;
+  history.push('/signup-form');
+}
+
+render() {
+  return (
+    <React.Fragment>
+      <Title />
+      <Header title="Login to save your data!" />
+      <StyledForm>
+        <StyledLabel> Username* </StyledLabel>
+        <StyledInput
+          type="text"
+          name="username"
+          placeholder="username"
+          value={this.state.username}
+          onChange={e => this.setState({
+            username: e.target.value,
+          })
             }
-          />
-          <StyledLabel> Password* </StyledLabel>
-          <StyledInput
-            type="password"
-            name="password"
-            placeholder="password"
-            value={this.state.password}
-            onChange={e => this.setState({
-              password: e.target.value,
-            })
+        />
+        <StyledLabel> Password* </StyledLabel>
+        <StyledInput
+          type="password"
+          name="password"
+          placeholder="password"
+          value={this.state.password}
+          onChange={e => this.setState({
+            password: e.target.value,
+          })
             }
-          />
+        />
 
-          <StyledP>
-            <StyledText>First time around here? </StyledText>
-            <StyledSignUp> Sign up</StyledSignUp>
-          </StyledP>
-        </StyledForm>
+        <StyledP>
+          <StyledText>First time around here? </StyledText>
+          <StyledSignUp onClick={this.goSignUp}> Sign up</StyledSignUp>
```

Comment:
This is in the login page, it should be called StyledLogin - you can extend the StyledSign in the style page but call it StyledLogin.